README.md

```diff
@@ -7,7 +7,14 @@ reports for WHO's *Argus* system.
 R libraries are managed by `packrat`.
 
 ## Other requirements
-Browser (e.g. chrome) for downloading the svg images for the report. In order to avoid loading `plotly` dependencies with the report the plots are rendered and saved using Plotly.downloadImage() via htmlwidgets::onRender() (see ?plotly::export for more information). This requires a web browser, with browser's download setting location set to `assets` folder.
+1. Browser (e.g. chrome) for downloading the svg images for the report. In order to avoid loading `plotly` dependencies with the report the plots are rendered and saved using `Plotly.downloadImage()` via `htmlwidgets::onRender()` (see `?plotly::export` for more information). This requires a web browser, with browser's download setting location set to `assets` folder.
```

Comment:
1. how about we say that this has to be chrome?
2. I suggest make `assets` location more precise (e.g. `[PROJECT_DIRECTORY]/assets`)